git-prompt.sh

```diff
@@ -120,6 +120,10 @@ __posh_git_ps1 ()
             ;;
     esac
     local gitstring=$(__posh_git_echo)
+	#same as: removing everything after   $BeforeBackgroundColor$BeforeForegroundColor$BeforeText 
+	ps1pc_prefix="${ps1pc_prefix%%'\[\e[1;33m\] ['*}"
```

Comment:
This is a fragile workaround. This will break if someone has that color sequence as prefix.